conventions.md

```diff
@@ -52,3 +52,19 @@ For integer types, they MUST be given in the following format:
 Implementors MAY treat fields without known sign to be unsigned - this can be the case for length fields for instance. It is unknown whether the original executable would handle them correctly.
 
 Unless otherwise noted, integer values are stored in [little-endian](http://en.wikipedia.org/wiki/Endianness#Little-endian) format.
+
+#### Fixed point numbers
+
+Due hardware limitations and slow perfomance of float point operations, System Shock uses fixed point number represenation of real data type.
+Usually engine uses [Q15.16](https://en.wikipedia.org/wiki/Q_(number_format)) format (1 bit for sign, 15 bits for integer part, 16 bits for fractional):
+
+    0x 0 000 0000
```

Comment:
Specifying it as "0x" I'd interpret these zeroes as hexadecimal, and (with a pedantic eye), I'd then think that the highest nibble (4 bits) would indicate the sign.
Perhaps expand this to ```0 - 000 0000 0000 0000 0000 0000 0000 0000 - 0000 0000 0000 0000 0000 0000 0000 0000b```?